docs/api.md

```diff
@@ -400,6 +399,21 @@ See the `bittorrent-protocol`
 [extension api docs](https://github.com/feross/bittorrent-protocol#extension-api) for more
 information on how to define a protocol extension.
 
+## `torrent.on('infoHash', function () {})`
+
+Emitted when the info hash of the torrent has been determined. Can only be accessed synchronously as client.add calls onTorrent callback after 'infoHash' event has been emitted.
```

Comment:
I don't think we need the extra text. It makes it more confusing IMO. It's already clear from the fact that this is an event that it's async.